doc/holochain_101/src/bridging.md

```diff
@@ -56,28 +56,23 @@ pub fn handle_sample_function(some_param: String) -> ZomeApiResult<Address> {
     // do something here
 }
 
-define_zome! {
-    entries: []
-
-    init: || { Ok(()) }
-
-    validate_agent: |validation_data : EntryValidationData::<AgentId>| {
+#[zome]
+mod zome {
+    #[init]
+    fn init() -> ZomeApiResult<()> {
         Ok(())
     }
 
-    functions: [
-        sample_function: {
-            inputs: |some_param: String|,
-            outputs: |result: ZomeApiResult<Address>|,
-            handler: handle_sample_function
-        }
-    ]
+    #[validate_agent]
```

Comment:
Is there a default `#[validate_agent]` implementation so that it's not required to define this to just get this simple `Ok(())` result?